x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/layer/compatible_layer.tsx

```diff
@@ -0,0 +1,432 @@
+              'Job ID must be no more than {maxLength, plural, one {# character} other {# characters}} long.',
+            values: {
+              maxLength: JOB_ID_MAX_LENGTH,
+            },
+          })
+        );
+      } else {
+        const resp = await ml.jobs.jobsExist([jobId]);
+        if (resp[jobId].exists) {
+          setJobIdValid(
+            i18n.translate('xpack.ml.newJob.wizard.validateJob.jobNameAlreadyExists', {
+              defaultMessage:
+                'Job ID already exists. A job ID cannot be the same as an existing job or group.',
+            })
+          );
+        }
+      }
+
+      if (validationResults.contains('bucket_span_invalid')) {
+        setBucketSpanValid(invalidTimeIntervalMessage(bucketSpan));
+      }
+      setState(STATE.DEFAULT);
+    },
+    500,
+    [jobId, bucketSpan]
+  );
+
+  return (
+    <>
+      {state !== STATE.SAVE_SUCCESS && state !== STATE.SAVING ? (
+        <>
+          <EuiFlexGroup gutterSize="s" data-test-subj="mlLensLayerCompatible">
+            <EuiFlexItem grow={false}>
+              <EuiText size="s">
+                <EuiIcon type="checkInCircleFilled" color="success" />
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText size="s">
+                {layer.jobWizardType === CREATED_BY_LABEL.MULTI_METRIC ? (
+                  <FormattedMessage
+                    id="xpack.ml.embeddables.lensLayerFlyout.createJobCalloutTitle.multiMetric"
+                    defaultMessage="This layer can be used to create a multi-metric job"
+                  />
+                ) : (
+                  <FormattedMessage
+                    id="xpack.ml.embeddables.lensLayerFlyout.createJobCalloutTitle.singleMetric"
+                    defaultMessage="This layer can be used to create a single metric job"
+                  />
+                )}
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="m" />
+          <EuiForm>
+            <EuiFormRow label="Job ID" error={jobIdValid} isInvalid={jobIdValid !== ''}>
+              <EuiFieldText
+                data-test-subj={`mlLensLayerJobIdInput_${layerIndex}`}
+                value={jobId}
+                onChange={(e) => {
+                  setJobId(e.target.value);
+                  setState(STATE.VALIDATING);
+                }}
+              />
+            </EuiFormRow>
+
+            <EuiSpacer size="s" />
+            <EuiAccordion
+              data-test-subj={`mlLensLayerAdditionalSettingsButton_${layerIndex}`}
+              id="additional-section"
+              buttonContent={i18n.translate(
+                'xpack.ml.embeddables.lensLayerFlyout.createJobCallout.additionalSettings.title',
+                {
+                  defaultMessage: 'Additional settings',
+                }
+              )}
+            >
+              <EuiSpacer size="s" />
+              <EuiFormRow
```

Comment:
I wonder if the bucket span setting should be moved out of advanced settings to make it clear what is being used? `15m` is set as the default, which will often be too short.

Reply:
I'll add in the bucket span estimator button in a follow up PR, I think we could also automatically run it when showing the flyout.
I could also make the change suggested here, but it might not be necessary if the generated bucket span is correct.